src/client/Microsoft.Identity.Client/MsalErrorMessage.cs

```diff
@@ -117,6 +117,7 @@ internal static class MsalErrorMessage
         public const string BrokerResponseError = "Broker response returned error: ";
         public const string CannotInvokeBroker = "MSAL cannot invoke the broker. The Authenticator App (Broker) may not be installed on the user's device or there was an error invoking the broker. " +
             "Check logs for more details and see https://aka.ms/msal-brokers. ";
+        public const string NullIntentReturnedFromBroker = "Broker returned a null intent. Check the Xamarin Android app settings and logs for more information.";
```

Comment:
add an aka.ms link here w/more info. what does the dev need to do to be able to resolve this error on their own? what specifically should be checked in the app settings and logs?

Reply:
ok, ill add to the android wiki page when i create

Reply:
you can make the page anytime and add the link here. just put a disclaimer at the top that the feature is not available yet. i think we did this w/ios broker. @jmprieur to weigh in.

---
In reply to: [355604318](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1510#discussion_r355604318) [](ancestors = 355604318)